cypress-E2E-tests/cypress/integration/asset_properties_widget.cy.ts

```diff
@@ -531,6 +531,59 @@ describe('Asset Properties Widget: Configuration/View screen tests', function ()
     cy.deleteCard();
   });
 
+  // This function is being used to check File validation error message
+  function verifyFileValidationError(errorText) {
+    cy.get('c8y-drop-area p').should('contain.text', errorText);
+  }
+
+  // This function is being used to select a file for an asset property
+  // Note: type is an optional parameter. If you are selecting a file for complex property then mention the type as 'complex'.
+  function selectFile(propertyName, fileName, type?) {
+    if (type) {
+      cy.get('c8y-asset-properties-item formly-field label')
+        .contains(propertyName)
+        .find("~c8y-drop-area div~input[type='file']")
+        .selectFile(`cypress/fixtures/${fileName}`, { force: true });
+    } else
+      cy.get('c8y-asset-properties-item formly-field p')
+        .contains(propertyName)
+        .find("~formly-field div~input[type='file']")
+        .selectFile(`cypress/fixtures/${fileName}`, { force: true });
+  }
+
+  // Verify file size and file type validations
+  it('TC_Asset_Properties_Widget_view_011', () => {
+    const propKey = 'file';
+    const invalidFileName = 'Image SAG LOGO.png';
+    const invalidFileSize = 'image1.jpg';
+    const errorMessage = 'The selected file is not supported.';
+    const invalidFileSizeErrorMessage= 'The selected file is too large. The size limit is 102.4 kB.'
+    const editPropertyCancelButton = "button[title='Cancel']";
+    const complexPropertyTitle = 'ComplexProperty';
+    cy.selectAssetPropertyAndSave('Test Asset3', propKey);
+    cy.clickPropertyEditButton('File');
+    cy.get('.dlt-c8y-icon-plus-square').click();
+    selectFile('File', invalidFileSize);
+    verifyFileValidationError(invalidFileSizeErrorMessage);
+    selectFile('File', invalidFileName);
+    verifyFileValidationError(errorMessage);
+    cy.get(editPropertyCancelButton).click();
+    cy.get(asset_properties_widget_elements.settingsButton).click();
+    cy.get(asset_properties_widget_elements.editWidgetButton).click();
+    // added wait to resolve flakyness after selecting editWidget its take few second to get properties.
+    cy.wait(1000)
```

Comment:
Use intercepts instead of arbitrary wait if the time taken to get the properties is due to a backend call.

Reply:
added intercept instead timeOut.